browser/src/control/Control.JSDialog.js

```diff
@@ -400,7 +400,7 @@ L.Control.JSDialog = L.Control.extend({
 			focusWidget = instance.init_focus_id ? instance.container.querySelector('[id=\'' + instance.init_focus_id + '\']') : null;
 		}
 
-		if (focusWidget && document.activeElement !== focusWidget) {
+		if (focusWidget && document.activeElement !== focusWidget && instance.canHaveFocus) {
```

Comment:
Is it fine to just not do anything in this case? You don't mention in a comment or in the commit message if you assume that the outer element has the focus already at this point.

Reply:
I think if "`instance.canHaveFocus`" is false then we musn't check for focusables. But maybe @eszkadev has a better idea here? This part may need some refactoring.

Reply:
Because that if statement conflicts with `instance.canHaveFocus` property.